### ISSUE TITLE
ui: app menu - improve

### ISSUE THREAD

**Issue by dsebastien** [NationalBankBelgium/stark#206]
Goal: in addition to what it already has, the menu entry interface should have:
* [ ] icon
* [ ] tooltip
* [ ] hotkey/key combination

Support for hotkeys would be good for UX & accessibility
